clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -9924,12 +9917,56 @@ Value *CodeGenFunction::EmitAArch64SVEBuiltinExpr(unsigned BuiltinID,
       // immediate requires more than a handful of bits.
       *Result = Result->extOrTrunc(32);
       Ops.push_back(llvm::ConstantInt::get(getLLVMContext(), *Result));
+      continue;
+    }
+
+    if (IsTupleGetOrSet) {
+      Ops.push_back(EmitScalarExpr(E->getArg(i)));
```

Comment:
This is the same as `Ops.push_back(Arg);` ? And if so, you can combine it with the case below.